Gebruikersdocumentatie: uitbreiding installatie-instructies en TLS (#107)
kiesraad/abacus-documentatie: gebruikersdocumentatie/src/installatie/installeren-starten/netwerk.md

# Netwerk inrichten

> Voordat je Abacus installeert is het belangrijk dat het netwerk goed is ingericht. Dit netwerk kun je op dezelfde manier opzetten als bij OSV2020, en je kunt Abacus en OSV2020 dan ook op dezelfde computer draaien.

Reviewer: Dit was het geval omdat Abacus een andere poort gebruikte. Dat deel van de config is aangepast en niet getest dus ik zou het niet aanraden.

> Voordat je Abacus installeert is het belangrijk dat het netwerk goed is ingericht. Dit netwerk kun je op dezelfde manier opzetten als bij OSV2020, en je kunt Abacus en OSV2020 dan ook op dezelfde computer draaien.

Reviewer: ```suggestion
Voordat je Abacus installeert is het belangrijk dat het netwerk goed is ingericht. De aansluit- en gebruiksvoorwaarden van de Kiesraad bevatten de regels hiervoor. Dit netwerk kun je op dezelfde manier opzetten als bij OSV2020.
```

Author: Ik wil niet op te veel plekken verwijzen naar links die later weer kapotgaan. Dat wil ik alleen in de systeemvereisten doen.

Reviewer: Suggestie aangepast - hier verwijzen naar het juiste document is weer een extra zetje en hulp bij het correct inrichten.

> Voordat je Abacus installeert is het belangrijk dat het netwerk goed is ingericht. De aansluit- en gebruiksvoorwaarden van de Kiesraad bevatten de regels hiervoor. Dit netwerk kun je op dezelfde manier opzetten als bij OSV2020.

Reviewer 2: Het zou mooi zijn als we kunnen linken naar de aansluit- en gebruiksvoorwaarden, maar lijkt erop dat we geen stabiele link hebben, omdat de voorwaarden steeds binnen de toolkit van verkiezing gepubliceerd worden.

Author: Klopt, en daarom wil ik hier (nog) geen link in zetten. Ook link ik elders al naar de aansluit- en gebruiksvoorwaarden, dus dit dubbel doen en dan riskeren dat ik het aanpassen van de link ergens vergeet lijkt me ook niet handig. Wel met je eens dat dit niet ideaal is.

Author: Voorstel om in ieder geval aan te geven waar de aansluit- en gebruiksvoorwaarden te vinden zijn:

```suggestion
Voordat je Abacus installeert is het belangrijk dat het netwerk goed is ingericht. De regels hiervoor staan in de aansluit- en gebruiksvoorwaarden, die je vindt in de toolkit van de verkiezing op de website van de Kiesraad. Dit netwerk kun je op dezelfde manier opzetten als bij OSV2020.
```

Je installeert Abacus op één computer die dient als server. De andere computers zijn clients en maken via de browser verbinding met de Abacus-server. Abacus mag alleen gebruikt worden in een netwerk dat niet verbonden is met het internet. Dit noemen we ook wel een *airgapped* netwerk.

## Benodigdheden

- Een hub of netwerkapparaat
- Een server en een aantal clients
- Voldoende ethernetkabels voor alle apparaten

> - Extra (unmanaged) netwerkswitches om meer apparaten te kunnen aansluiten (optioneel)

Reviewer: ```suggestion

```
Hier kom je vlot in conflict met de aansluit en gebruiksvoorschriften, ik zou deze zin weglaten.
